charts/spire/charts/spire-server/values.yaml

```diff
@@ -182,15 +182,21 @@ upstreamAuthority:
         key: ""
         bundle: ""
   certManager:
-    enabled: false
+    enabled: true
```

Comment:
Can we leave the default to false? So we still have same default behavior for consumers of the chart.